INFO     root:test_User_Interface.py:1210 Reset button clicked successfully.